tests/test_parser.py

```diff
@@ -257,6 +259,98 @@ def test_convert_to_date_cast_result_to_date(self, mocked_locale,
                          expected_output)
 
 
+class Test_bs_safeget(unittest.TestCase):
+    def test_bs_safeget_return_expected_text(self):
+        mocked_soup = MagicMock()
+        selector = 'some_tag'
+        mocked_soup.select.return_value = ['some text']
+        self.assertEqual(bs_safeget(mocked_soup, selector), 'some text')
+
+    def test_bs_safeget_return_false_when_selector_is_missing(
```

Comment:
test name is wrong. selector = h1 in your test, not missing. What exactly is tested here?

Reply:
renamed to test_bs_safeget_return_false_when_selectedElems_is_empty